x/group/keeper/msg_server.go

```diff
@@ -840,18 +823,24 @@ func (k Keeper) LeaveGroup(goCtx context.Context, req *group.MsgLeaveGroup) (*gr
 		return nil, err
 	}
 
+	if updatedWeight.IsZero() {
+		return nil, sdkerrors.ErrInvalidRequest.Wrap("last member cannot leave group")
+	}
```

Comment:
Simulations panicked because there was a group with TotalWeight == 0.

Should we ever allow that?

The scenario is the last member of a group does a LeaveGroup. I added a logic here to disallow this member leaving, but would like to double check.

cc @cmwaters @blushi 

The alternative would be to allow groups with TotalWeight == 0.